build.gradle

```diff
@@ -158,9 +158,9 @@ subprojects {
                 protobuf_plugin: 'com.google.protobuf:protobuf-gradle-plugin:0.8.0',
                 protobuf_util: "com.google.protobuf:protobuf-java-util:${protobufVersion}",
 
-                netty: 'io.netty:netty-codec-http2:[4.1.4.Final]',
+                netty: 'io.netty:netty-codec-http2:[4.1.6.Final-SNAPSHOT]',
                 netty_epoll: 'io.netty:netty-transport-native-epoll:4.1.4.Final' + epoll_suffix,
-                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork19',
+                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork22',
```

Comment:
@ejona86 no it's not required. will remove the dependency changes once we are using 4.1.6. I just set it to Fork22 so that the tests pass (there was some issue with our unit tests and TLS). IIRC that also prevented us from upgrading to 4.1.5.